README.md

```diff
@@ -3,3 +3,35 @@
 This is my CUDA implementation of [Numenta's Spatial Pooling algorithm](https://www.frontiersin.org/articles/10.3389/fncom.2017.00111/full).
 
 My goal starting this project was to create an efficient CUDA backend for the entire [Numenta's Platform for Intelligent Computing](https://github.com/numenta/nupic) (NuPIC). Unfortunately, I don't have the time to continue at this moment, but I am open to starting an OSS project. If you are interested, do not hesitate to contact me!
```

Comment:
If you don't mind, please remove the last, personal sentence: "Unfortunately,..."